src/cpp/filter_image.hpp

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+/*
+    Here are the image filters
+*/
+
```

Comment:
This file is called from process_image.cpp only in iOS so that there is not a problem with the openCV library

Reply:
Okay, thank you for explaining the use of this file!

Reply:
This all looks good! Was openCV not working quite right in iOS before then?

Reply:
It was working in iOS, but without this file to separate it, Android would not work because it will say “opencv not found”

Reply:
Basically this is needed until you get openCV to work on Android, then (if you want) this function can go back into process_image.cpp